SPECS/libwebp/libwebp.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "libwebp-1.0.0.tar.gz": "c5c5ebf979543ac1f3348df8f6245262abd787a147b9632c880d92bfc38dbbeb"
+     "libwebp-1.0.3.tar.gz": "082d114bcb18a0e2aafc3148d43367c39304f86bf18ba0b2e766447e111a4a91"
  }
-}
```

Comment:
Nit: unnecessary new line - I think we don't have this in other/most signature JSONs. I don't think it makes any difference for our needs, though.

Reply:
If I am not mistaken, files without the newline at the end are not POSIX compliant: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206